RecoBTag/ONNXRuntime/python/pfUnifiedParticleTransformerAK4V1_cff.py

```diff
@@ -0,0 +1,29 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoBTag.FeatureTools.pfUnifiedParticleTransformerAK4TagInfos_cfi import pfUnifiedParticleTransformerAK4TagInfos
+from RecoBTag.ONNXRuntime.pfUnifiedParticleTransformerAK4JetTags_cfi import pfUnifiedParticleTransformerAK4JetTags
+from RecoBTag.ONNXRuntime.pfUnifiedParticleTransformerAK4V1DiscriminatorsJetTags_cfi import pfUnifiedParticleTransformerAK4V1DiscriminatorsJetTags
+
+#
+pfUnifiedParticleTransformerAK4V1TagInfos = pfUnifiedParticleTransformerAK4TagInfos.clone()
```

Comment:
Does it need to set `fix_lt_sorting` back to `False` for v1?

Reply:
From the regression plots made, it seems UParTv1 regression has not been affected by `fix_lt_sorting` so I believe we can leave it like that

Reply:
If the plan is to keep the fix_lt_sorting True for all UParT models, then we could simply remove this flag and implement the fix permanently